@stellar/typescript-wallet-sdk-soroban/test/helpers.test.ts

```diff
@@ -323,3 +330,254 @@ describe("scValByType should render expected common types", () => {
     expect(parsedVoid).toEqual(null);
   });
 });
+
+describe("getInvocationDetails for a Soroban Authorized Invocation tree", () => {
+  const randomKey = (): string => {
+    return Keypair.random().publicKey();
+  };
+
+  const makeInvocation = (
+    contract: Contract,
+    name: string,
+    ...args: any[]
+  ): xdr.SorobanAuthorizedFunction => {
+    return xdr.SorobanAuthorizedFunction.sorobanAuthorizedFunctionTypeContractFn(
+      new xdr.InvokeContractArgs({
+        contractAddress: contract.address().toScAddress(),
+        functionName: name,
+        args: args.map((arg) => nativeToScVal(arg)),
+      }),
+    );
+  };
+
+  const invoker = randomKey();
+
+  const [nftContract, swapContract, xlmContract, usdcContract] = [
+    1, 2, 3, 4,
+  ].map(() => {
```

Comment:
this might make a nice test helper!

Reply:
Nice catch! I've just extracted a few test helpers here: https://github.com/stellar/typescript-wallet-sdk/pull/128/commits/6e8aae9e77165fb334bccba7cf7e5f049c32c522